lib/serializer/CTypeList.h

```diff
@@ -109,12 +109,7 @@ class DLL_LINKAGE CTypeList: public boost::noncopyable
 
 		return ptr;
 	}
-	CTypeList &operator=(CTypeList &)
-	{
-		// As above.
```

Comment:
which "above" does it refer to?

Reply:
It seems to be lost in ages. But this operator will always assert, so, I think deleting it instead is better, because it just not compile if requested instead of runtime error.

Reply:
thanks! deleting is fine, I just couldn't understand what this comment means.